src/components/ProfilePage/ProfilePage.jsx

```diff
@@ -397,7 +397,7 @@ function ProfilePage() {
                             <p>Phone Number: {profile.phone_number}</p>
                             <p>Description: {profile.description}</p>
                             <p>ski or snow: {profile.ski_or_snow}</p>
-                            <p>Coach Levels: {profile.coach_level}</p>
+                            <p>Coach Levels: {profile.coach_level != 'Unqualified' ? profile.coach_level : 'Uncertified'}</p>
```

Comment:
this line of code checks if the value from the database for coach_level is not equal to "Unqualified". If DB value from GET route show coach_level is "Unqualified", then the DOM will render "Uncertified"